doc/10-icinga-template-library.md

```diff
@@ -2799,6 +2799,30 @@ smart_attributes_config_path    | **Required.** Path to the smart attributes con
 smart_attributes_device         | **Required.** Device name (e.g. /dev/sda) to monitor.
 
 
+#### smart-advanced <a id="plugin-contrib-command-smart-advanced"></a>
+
+The [check_smart](https://www.claudiokuenzler.com/monitoring-plugins/check_smart.php) plugin uses the `smartctl` command from the `smartmontools` package to monitor SMART values of physical drives. Hard, solid state and NVMe drives are supported. Both Direct drive access or drive behind a storage controller (such as RAID) are supported.
+
+Either smart_device (-d) or smart_global (-g) is required. smart_interface (-i) is always required.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                   | Description
+-----------------------|-----------------------------------------------------------------------
+smart_device           | **Required.** Path to physical block device to be SMART monitored, e.g. /dev/sda.
+smart_interface        | **Required.** Drive's interface type, must be one of: auto, ata, scsi, nvme, 3ware,N, areca,N, hpt,L/M/N, cciss,N, megaraid,N.
+smart_global           | **(Required.)** A glob expression of physical devices to be monitored, eg -g "/dev/sd[a-z]".
```

Comment:
Why is this instance of required in parenthesis? From the statement above the table I'd say `smart_device` and `smart_global` should look the same here.

Also not totally sure about what's the best way to document "one out of two is required" here, maybe someone else has an idea? How is this done for other plugins? (I'd assume this is not the first time this happens)

Reply:
Because `smart_device` uses the `-d` parameter in the background while `smart_global` uses `-g`. These are different implementations. Don't know how to "style" it that it makes sense.

Additional description should probably note "Either one of smart_device or smart_global is required - they can not coexist". 

More information about this can be found in the official plugin documentation.

Reply:
@Al2Klimov Can you maybe comment on how to handle this? I'd rather avoid introducing new and potentially unclear notation like "(Required)". I have a slight tendency saying as each variable seen individually is not strictly required (and they even conflict each other), labeling them as optional and the notice above the table is good enough.

Reply:
The latter sound good, but please have a look how this is handled for other check commands.

Reply:
Do you happen to know one where this is the case? Took a quick look at the docs and didn't find one straightaway.

Reply:
Then it seems to be the first time. I'd note the requirement not above the table, but next to each param.